### PR TITLE
Validation of total attachments.

### PR DESCRIPTION
**Pull request by ivanelrey**
Issue: #12 
Example:

  validates :files, limit { min: 1, max: 5 }

  An error will occur if total number of attached files are below 1 or greater that 5.

Error message:
  limit_out_of_range: "total number is out of range"

Also max and min can be used in the error message.
  limit_out_of_range: "total number should be between %{min} and %{max}"

 On branch add-limit-validation
 Changes to be committed:
	modified:   ../config/locales/en.yml
	modified:   ../lib/active_storage_validations.rb
	modified:   active_storage_validations_test.rb
	new file:   dummy/.byebug_history
	modified:   dummy/app/models/project.rb

 Untracked files:
	../lib/active_storage_validations/limit_validator.rb

### CODE REVIEW

File: test/dummy/.byebug_history [hunk 1]
```diff
@@ -0,0 +1,40 @@
+continue
```
Comment (igorkasyanchuk) [inline test/dummy/.byebug_history]: please add to gitignore

**Comment by igorkasyanchuk** [comment]
I've only one comment.
Please fix and I'll merge, looks very good and useful

**Comment by igorkasyanchuk** [comment]
@ivanelrey  CC

**Comment by ivanelrey** [comment]
@igorkasyanchuk Done. Thanks a lot.

**Comment by igorkasyanchuk** [comment]
new version just released. thanks

**Pull request merged**